### DISCUSSION TITLE
Improve UX by summarizing cart total at the end of the checkout form

### DISCUSSION THREAD

**Discussion by passatgt** [woocommerce/woocommerce#45082] [WooCommerce, Blocks, and Site Editing]
At the moment the order total is displayed at the top of the checkout page(top right on desktop). Since the total can change when you select a shipping method, and in some cases additional fees are added based on billing details / payment methods(like COD fee), the customer doesn't really see that the order total changes unless they scroll up to the top, so i think it would be a good idea to create a new inner block that can summarize the order total. 

Couple of ideas regarding this:

On desktop view, this problem can be solved by fixing the right sidebar on scroll, so it's always visible. Maybe closing the order summary on scroll automatically. 
![image](https://user-images.githubusercontent.com/43415/222114879-79de7a10-72f3-430d-9405-d1e1e5c058e2.png)

On mobile, an additional order summary block above the place order button
![image](https://user-images.githubusercontent.com/43415/222116573-eb1e4d62-9e82-40fd-96f5-22bd20160c35.png)

Maybe change the place order label button dynamically to indicate the cart total
![image](https://user-images.githubusercontent.com/43415/222116918-cd3841ad-34ce-4cb7-9d9f-7a810bd9e3b3.png)

**Comment by opr** [comment]
Hey there @passatgt thank you for posting your suggestion!

I personally really like these suggestions, and I think having the price in the button is a good idea. I wonder if having an option to enable/disable it would be a good idea too.

> On mobile, an additional order summary block above the place order button

I wonder if increasing the scroll distance required to get to the place order button (even if by a few lines) would negatively affect conversions?

> On desktop view, this problem can be solved by fixing the right sidebar on scroll, so it's always visible

@nerrad mentioned there is a new sticky position option available for container blocks. This might be a good place to try that out, but it may require some structural changes to our blocks to make use of that.

cc @woocommerce/rubik to take a look at this discussion.

**Reply by passatgt** [reply]
> I wonder if increasing the scroll distance required to get to the place order button (even if by a few lines) would negatively affect conversions?

I was thinking of this being an optional inner block. So one could move it below the button or above the button, but the terms & conditions can be below it for example. Either way, it would be a nice option to have. 

Or, if we are doing sticky sidebar, maybe a sticky row at the top on mobile that can drop down? (this one more difficult due to theme compatibility issues)

**Reply by opr** [reply]
> I was thinking of this being an optional inner block. So one could move it below the button or above the button, but the terms & conditions can be below it for example. Either way, it would be a nice option to have.

That's a good idea too, I appreciate you sharing this with us.

> Or, if we are doing sticky sidebar, maybe a sticky row at the top on mobile that can drop down? (this one more difficult due to theme compatibility issues)

We already stick the `Proceed to Checkout` button to the bottom of the screen on the Cart block for mobile devices, so there is something in place for that. Maybe the order summary could stick to the top once the user has scrolled past the original one 🤔.

Thanks for the suggestions, I have reminded the team about this discussion, I think these will definitely require some design input before we can act on them.
